src/Generator.jl

```diff
@@ -25,23 +25,44 @@ end
 """
 $(SIGNATURES)
 
+Attempts to append to a file at `\$(root)/\$(filename)`. `f` will be called with
+file stream (see [`open`](http://docs.julialang.org/en/latest/stdlib/io-network/#Base.open)).
```

Comment:
Probably don't really need to link to `open` here since this is just a small internal function.